reference/conanfile/tools/cmake/cmaketoolchain.rst

```diff
@@ -159,15 +159,16 @@ CMake legacy generators (like ``cmake``, or ``cmake_paths``) with it.
 Using a custom toolchain file
 +++++++++++++++++++++++++++++
 
-There are two ways of providing a custom CMake toolchain file:
+There are two ways of providing a custom CMake toolchain files:
 
-- The ``conan_toolchain.cmake`` file can be completely skipped and replaced by a user one, defining the
-  ``tools.cmake.cmaketoolchain:toolchain_file=<filepath>`` configuration value
+- The ``conan_toolchain.cmake`` file can be completely skipped and replaced by an user one, defining the
+  ``tools.cmake.cmaketoolchain:toolchain_file=["<filepath>"]`` configuration list or even appending to any
```

Comment:
But `tools.cmake.cmaketoolchain:toolchain_file` is not a list right? The one that is a list is: `tools.cmake.cmaketoolchain:user_toolchain`

Reply:
yes, ``user_toolchain`` is a list, this one must be a single value, as it is the cli arg -DCMAKE_TOOLCHAIN_FILE

Reply:
oh, sure. Didn't see the whole field...